meetings/working-groups/extensions/Extension-API-docs.md

```diff
@@ -0,0 +1,150 @@
+}
+```
+
+The `source` parameter is referred to as the *receiver parameter*, or *receiver*. The *receiver parameter* may be an instance, as shown above, or it may be a type, as in the following:
+
+```csharp
+extension<T>(IEnumerable<T>)
+{
+    public static IEnumerable<T> Create(int size, Func<int, T> factory);
+}
+```
+
+This presentation enables the following:
+
+- Readers see the new extension syntax when consuming new extensions, driving awareness and adoption.
+- Readers can quickly distinguish a new-style extension (noted by the receiver parameter), and existing extension methods (noted by the `this` modifier on the first parameter).
+- The `extension` container indicates that the member is an extension member.
+- The parameter on the `extension` node indicates the type extended, and provides a key to know if the member is intended to extend an instance or extend the type.
+- The prototype only includes the `static` modifier if it would be present in source, as in an operator declaration:
+  ```csharp
+  extension<T>(IEnumerable<T> source)
+  {
+      public static IEnumerable<T> operator + (IEnumerable<T> left, IEnumerable<T> right);
+  }
+  ```
+
+### Extension members in the extended type's page
+
+The API docs build system generates the section on the type page for the extended type that lists all extension members. This section should have sub-sections for *extension methods*, *extension properties*, and *extension operators*. Extension indexers should follow the format for indexers, and be listed as an `Item[]` property. There isn't a `this` modifier on the first parameter. In fact, the receiver is declared on the extension, not the member. The prototypes in this section should expand to show the `extension` container, as follows:
+
+- The prototypes are displayed as described in the previous section.
+- Otherwise, the format is consistent with the current format:
+  - Each overload has a separate entry in the section.
+  - The signatures indicate if the method is a specialization (`Sum(this IEnumerable<int>)`) or generic (`Where<T>(this IEnumerable<T> source, Func<T, bool> predicate)`).
+  - The section lists all extensions, from all extending types. They are sorted alphabetically, as proposed for current extension methods.
+  - The signature does not show any indication of the extending type.
+
+### Extension class page
+
+The page for the class containing extensions will need only minimal updates in how extension members are displayed.
+
+- The TOC node for the class will have additional nodes for **Properties** (including indexers), and **operators**. Classes already support this, so it should already work. Note that the node for methods displays *method groups*, not *individual overloads*. That should remain.
+- The page should also have sections for **Properties**, and **operators**.
+- The prototypes for extensions should be displayed as shown [above](#extension-member-prototypes).
+
+### Extension member page
```

Comment:
This will display the static class where the extension is declared, right? That's the type name needed for disambiguation. Will it be displayed somewhere else as well?